README.md

```diff
@@ -50,91 +50,69 @@ GPUStack is an open-source GPU cluster manager for running AI models.
 
 ## Installation
 
-### Linux or macOS
+### Linux
 
-GPUStack provides a script to install it as a service on systemd or launchd based systems with default port 80. To install GPUStack using this method, just run:
+If you are using NVIDIA GPUs, ensure [Docker](https://docs.docker.com/engine/install/) and [NVIDIA Container Toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html) are installed on your system. Then, run the following command to start the GPUStack server.
 
 ```bash
-curl -sfL https://get.gpustack.ai | sh -s -
+docker run -d --name gpustack \
+      --restart=unless-stopped \
+      --gpus all \
+      --network=host \
+      --ipc=host \
+      -v gpustack-data:/var/lib/gpustack \
+      gpustack/gpustack
 ```
 
-### Windows
+For more details on the installation or other GPU hardware platforms, please refer to the [Installation Documentation](installation/installation-requirements.md).
 
-Run PowerShell as administrator (**avoid** using PowerShell ISE), then run the following command to install GPUStack:
-
-```powershell
-Invoke-Expression (Invoke-WebRequest -Uri "https://get.gpustack.ai" -UseBasicParsing).Content
-```
-
-### Other Installation Methods
-
-For manual installation, docker installation or detailed configuration options, please refer to the [Installation Documentation](https://docs.gpustack.ai/latest/installation/installation-script/).
-
-## Getting Started
-
-1. Run and chat with the **llama3.2** model:
+After the server starts, run the following command to get the default admin password:
 
 ```bash
-gpustack chat llama3.2 "tell me a joke."
+docker exec gpustack cat /var/lib/gpustack/initial_admin_password
 ```
 
-2. Run and generate an image with the **stable-diffusion-v3-5-large-turbo** model:
+Open your browser and navigate to `http://your_host_ip` to access the GPUStack UI. Use the default username `admin` and the password you retrieved above to log in.
```

Comment:
After the Linux session, mention macOS & Windows and guide users to https://docs.gpustack.ai/latest/installation/desktop-installer/